src/mock/api/veilarbperson.ts

```diff
@@ -1,9 +1,15 @@
 import { rest } from 'msw';
 import { RequestHandlersList } from 'msw/lib/types/setupWorker/glossary';
-import { hentMalform } from '../api-data';
+import { hentMalform, hentMalformV2 } from '../api-data';
 
 export const veilarbpersonHandlers: RequestHandlersList = [
 	rest.get('/veilarbperson/api/person/:fnr/malform', (req, res, ctx) => {
 		return res(ctx.delay(500), ctx.json(hentMalform()));
 	})
 ];
+
+export const veilarbpersonHandlersV2: RequestHandlersList = [
```

Comment:
Tenker det er best å slå sammen alle handlers for veilarbperson i 1 liste

Reply:
Utført